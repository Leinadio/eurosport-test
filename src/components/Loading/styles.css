.loading-text {
    font-family: Arial;
}