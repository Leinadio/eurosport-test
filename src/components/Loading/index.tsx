import React from 'react';
import './styles.css'

export default function Loading(): JSX.Element {
  return (
    <p className="loading-text">loading...</p>
  )
}
