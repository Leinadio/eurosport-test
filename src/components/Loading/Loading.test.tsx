import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './'

describe('Loading', function () {
  it('should loading', () => {
    const tree = renderer
      .create(<Loading />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});