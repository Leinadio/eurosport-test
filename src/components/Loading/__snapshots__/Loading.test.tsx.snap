// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should loading 1`] = `
<p
  className="loading-text"
>
  loading...
</p>
`;
